test/models/role_test.rb

```diff
@@ -0,0 +1,28 @@
+require "test_helper"
+
+class RoleTest < ActiveSupport::TestCase
+  def setup
+    @organization = create(:organization)
+    @staff_account = create(:staff_account, organization: @organization)
+  end
+
+  test "assigns the admin role to staff_account within organization scope" do
+    @staff_account.add_role(:admin, @organization)
+
+    assert @staff_account.has_role?(:admin, @organization)
+  end
+
+  test "does not assign the admin role to staff_account outside organization scope" do
+    other_organization = Organization.create(name: "Other Organization")
+    @staff_account.add_role(:admin, other_organization)
+
+    refute @staff_account.has_role?(:admin, @organization)
+  end
+
+  test "removes the admin role from staff_account" do
+    @staff_account.add_role(:admin, @organization)
+    @staff_account.remove_role(:admin, @organization)
+
+    refute @staff_account.has_role?(:admin, @organization)
+  end
```

Comment:
I don't think you need these tests because you are testing the gem "rollify" which we assume is working well. Although it adds some extra layer of protection against bugs, it is often redundant and should trust our gems to work properly.

Reply:
Yeah fair enough. It was as much for my own apeasement that I had set it up correctly

Reply:
I think if we didn't have other test coverages that the chances of releasing code without the really critical rolify wouldn't be to common. If this was removed, then I suspect every (if not all) controller and system tests would break because we rely on it.

Accidental removals happen and I think that a sign of good test coverage is everything breaking if something like this happens. Let me know if that makes sense.

Reply:
yes that does make sense! I have removed them.